app/controllers/submissions_controller.rb

```diff
@@ -4,7 +4,7 @@ class SubmissionsController < BasePublicController
   skip_before_action :send_unstarted_claimants_to_the_start, only: [:show]
 
   def create
-    @form = journey::ClaimSubmissionForm.new(journey_session: journey_session)
+    @form = journey::ClaimSubmissionForm.new(journey_session: journey_session, params: params)
```

Comment:
can the check your answers page not be a `Form` like most of the other pages in the journey? that way you won't have to do this params business. I think if you think of the check your answers page as a form for `submitter_name` but with a whole load of read-only content ie the check answers bit

Reply:
👍🏽 yes - that does work - using a callback `check_your_answers_after_form_save_success` to create and save the `ClaimSubmissionForm`